### PR TITLE
Fix whitespace around opening `code` tag

### PR DESCRIPTION
**Pull request by NilsIrl**
A flaw should probably be made for that although, this problem seems to have mainly been introduced by the WYSIWYG editor from before and so no regressions should occur.

Similar problems exist around the closing `code` (and probably every possible and imaginable tag for that matter).

### CODE REVIEW

**Comment by NilsIrl** [comment]
See https://github.com/mdn/content/pull/4698

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 1]
```diff
@@ -345,9 +345,9 @@ <h2 id="Examples_2"><a id="Examples"></a> <a id="1037918"> Examples </a></h2>
 <h3 id="Creating_a_New_CRL"> <a id="1028724"> Creating a New CRL </a></h3>
 <p><a id="1034720"> This example creates a new CRL and importing it in to a Database in the specified directory: </a></p>
 
-<p><a id="1034722"> <code>crlutil -G -d </code><i>certdir</i><code> -n </code><i>cert-nickname</i><code> -c </code><i>crl-script-file</i></a></p>
+<p><a id="1034722"> <code>crlutil -G -d </code><i>certdir</i> <code>-n </code><i>cert-nickname</i> <code>-c </code><i>crl-script-file</i></a></p>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:348]:
```suggestion
<p><a id="1034722"> <code>crlutil -G -d</code> <i>certdir</i> <code>-n</code> <i>cert-nickname</i> <code>-c</code> <i>crl-script-file</i></a></p>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 2]
```diff
@@ -345,9 +345,9 @@ <h2 id="Examples_2"><a id="Examples"></a> <a id="1037918"> Examples </a></h2>
...
 <p>or</p>
-<p><code>crlutil -G -d </code><i>certdir</i><code> -n </code><i>cert-nickname</i><code> &lt;&lt;<i>EOF<br>
+<p><code>crlutil -G -d </code><i>certdir</i> <code>-n </code><i>cert-nickname</i> <code>&lt;&lt;<i>EOF<br>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:350]:
```suggestion
<p><code>crlutil -G -d</code> <i>certdir</i> <code>-n</code> <i>cert-nickname</i> <code>&lt;&lt;<i>EOF<br>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 3]
```diff
@@ -360,7 +360,7 @@ <h3 id="Listing_CRLs_in_a_Database"><a id="1034026"> Listing CRLs in a Database
 <p><a id="1037237"> <code>CN=NSS Test CA,O=BOGUS NSS,L=Mountain View,ST=California,C=US  CRL<br>
   CN=John Smith,O=Netscape,L=Mountain View,ST=California,C=US  CRL</code></a></p>
 <p>To view a particular CRL user should specify <i>-n nickname</i> parameter.<br>
-  <code>crlutil -L -d </code><i>certdir</i><code> -n </code><i> nickname</i></p>
+  <code>crlutil -L -d </code><i>certdir</i> <code>-n </code><i> nickname</i></p>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:363]:
```suggestion
  <code>crlutil -L -d</code> <i>certdir</i> <code>-n</code> <i>nickname</i></p>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 4]
```diff
@@ -376,16 +376,16 @@ <h3 id="Listing_CRLs_in_a_Database"><a id="1034026"> Listing CRLs in a Database
           Revocation Date: Wed Feb 23 12:08:40 2005</code></p>
 <h3 id="Deleting_CRL_from_a_Database"><a id="1028731"> Deleting CRL from a Database</a></h3>
 <p><a id="1035439"> This example deletes CRL from a database in the specified directory: </a></p>
-<p><a id="1035441"> <code>crlutil -D -n </code><i>nickname</i><code> -d </code><i>certdir</i> </a></p>
+<p><a id="1035441"> <code>crlutil -D -n </code><i>nickname</i> <code>-d </code><i>certdir</i> </a></p>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:379]:
```suggestion
<p><a id="1035441"><code>crlutil -D -n</code> <i>nickname</i> <code>-d</code> <i>certdir</i></a></p>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 5]
```diff
@@ -376,16 +376,16 @@ <h3 id="Listing_CRLs_in_a_Database"><a id="1034026"> Listing CRLs in a Database
...
 <h3 id="Importing_CRL_into_a_Database"><a id="1034020"> Importing CRL into a Database </a></h3>
 <p>This example imports CRL into a database: </p>
-<p><code>crlutil -I -i </code><i>crl-file</i><code> -d </code><i>certdir</i></p>
+<p><code>crlutil -I -i </code><i>crl-file</i> <code>-d </code><i>certdir</i></p>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:382]:
```suggestion
<p><code>crlutil -I -i</code> <i>crl-file</i> <code>-d</code> <i>certdir</i></p>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 6]
```diff
@@ -376,16 +376,16 @@ <h3 id="Listing_CRLs_in_a_Database"><a id="1034026"> Listing CRLs in a Database
...
 <p>File should has binary format of ASN.1 encoded CRL data. </p>
 <h3 id="Modifying_CRL_in_a_Database">Modifying CRL in a Database</h3>
 <p>This example modifies a new CRL and importing it in to a Database in the specified directory:</p>
-<p><code>crlutil -G -d </code><i>certdir</i><code> -n </code><i>cert-nickname</i><code> -c </code><i>crl-script-file</i></p>
+<p><code>crlutil -G -d </code><i>certdir</i> <code>-n </code><i>cert-nickname</i> <code>-c </code><i>crl-script-file</i></p>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:386]:
```suggestion
<p><code>crlutil -G -d</code> <i>certdir</i> <code>-n</code> <i>cert-nickname</i> <code>-c</code> <i>crl-script-file</i></p>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 7]
```diff
@@ -376,16 +376,16 @@ <h3 id="Listing_CRLs_in_a_Database"><a id="1034026"> Listing CRLs in a Database
...
 <p>or</p>
-<p><code>crlutil -M -d </code><i>certdir</i><code> -n </code><i>cert-nickname</i><code> &lt;&lt;<i>EOF<br>
+<p><code>crlutil -M -d </code><i>certdir</i> <code>-n </code><i>cert-nickname</i> <code>&lt;&lt;<i>EOF<br>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:388]:
```suggestion
<p><code>crlutil -M -d</code> <i>certdir</i> <code>-n</code> <i>cert-nickname</i> <code>&lt;&lt;<i>EOF<br>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html [hunk 8]
```diff
@@ -519,13 +519,13 @@ <h3 id="Script_Grammar">
 <a id="1041894">
     The script file grammar is as follows:</a>
 <pre><a id="1040388"><code>--&gt; </code><i>valuelist</i></a></pre>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html:521]:
```suggestion
<pre><a id="1040388"><code>--&gt;</code> <i>valuelist</i></a></pre>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html [hunk 9]
```diff
@@ -519,13 +519,13 @@ <h3 id="Script_Grammar">
...
-<pre><a id="1040390"><i>valuelist</i><code> --&gt; </code><i>value valuelist<br></i><code>               &lt;</code><i>null&gt;</i></a></pre>
-<pre><a id="1040394"><i>value</i><code> ---&gt; </code><i>key_value_pair<br></i>            <i>string</i></a></pre>
-<pre><a id="1040397"><i>key_value_pair</i><code> --&gt; </code><i>key </i><code>{ </code><i>valuelist</i><code> }</code></a></pre>
-<pre><a id="1040399"><i>key</i><code> --&gt; </code><i>string</i></a></pre>
-<pre><a id="1040401"><i>string</i><code> --&gt; </code><i>simple_string<br></i><code>            "</code><i>complex_string</i><code>"</code></a></pre>
-<pre><a id="1040404"><i>simple_string</i><code> --&gt; [^ \t\n\""{""}"]+ <br></code><var>(No whitespace, quotes, or braces.)</var></a></pre>
-<pre><a id="1040406"><i>complex_string</i><code> --&gt; ([^\"\\\r\n]|(\\\")|(\\\\))+ </code><var>(Quotes and<br>backslashes must be escaped with a backslash. A complex string must not<br>include newlines or carriage returns.)</var></a></pre>
+<pre><a id="1040390"><i>valuelist</i> <code>--&gt; </code><i>value valuelist<br></i><code>               &lt;</code><i>null&gt;</i></a></pre>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html:522]:
```suggestion
<pre><a id="1040390"><i>valuelist</i> <code>--&gt;</code> <i>value valuelist<br></i><code>               &lt;</code><i>null&gt;</i></a></pre>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html [hunk 10]
```diff
@@ -519,13 +519,13 @@ <h3 id="Script_Grammar">
...
+<pre><a id="1040394"><i>value</i> <code>---&gt; </code><i>key_value_pair<br></i>            <i>string</i></a></pre>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html:523]:
```suggestion
<pre><a id="1040394"><i>value</i> <code>---&gt;</code> <i>key_value_pair<br></i>            <i>string</i></a></pre>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html [hunk 11]
```diff
@@ -519,13 +519,13 @@ <h3 id="Script_Grammar">
...
+<pre><a id="1040397"><i>key_value_pair</i> <code>--&gt; </code><i>key </i><code>{ </code><i>valuelist</i> <code>}</code></a></pre>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html:524]:
```suggestion
<pre><a id="1040397"><i>key_value_pair</i> <code>--&gt;</code> <i>key </i><code>{ </code><i>valuelist</i> <code>}</code></a></pre>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html [hunk 12]
```diff
@@ -519,13 +519,13 @@ <h3 id="Script_Grammar">
...
+<pre><a id="1040399"><i>key</i> <code>--&gt; </code><i>string</i></a></pre>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html:525]:
```suggestion
<pre><a id="1040399"><i>key</i> <code>--&gt;</code> <i>string</i></a></pre>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html [hunk 13]
```diff
@@ -519,13 +519,13 @@ <h3 id="Script_Grammar">
...
+<pre><a id="1040401"><i>string</i> <code>--&gt; </code><i>simple_string<br></i><code>            "</code><i>complex_string</i><code>"</code></a></pre>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html:526]:
```suggestion
<pre><a id="1040401"><i>string</i> <code>--&gt;</code> <i>simple_string<br></i><code>            "</code><i>complex_string</i><code>"</code></a></pre>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html [hunk 14]
```diff
@@ -519,13 +519,13 @@ <h3 id="Script_Grammar">
...
+<pre><a id="1040404"><i>simple_string</i> <code>--&gt; [^ \t\n\""{""}"]+ <br></code><var>(No whitespace, quotes, or braces.)</var></a></pre>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html:527]:
```suggestion
<pre><a id="1040404"><i>simple_string</i> <code>--&gt; [^ \t\n\""{""}"]+<br> </code><var>(No whitespace, quotes, or braces.)</var></a></pre>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html [hunk 15]
```diff
@@ -519,13 +519,13 @@ <h3 id="Script_Grammar">
...
+<pre><a id="1040406"><i>complex_string</i> <code>--&gt; ([^\"\\\r\n]|(\\\")|(\\\\))+ </code><var>(Quotes and<br>backslashes must be escaped with a backslash. A complex string must not<br>include newlines or carriage returns.)</var></a></pre>
```
Comment (Ryuno-Ki) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_modutil/index.html:528]:
```suggestion
<pre><a id="1040406"><i>complex_string</i> <code>--&gt; ([^\"\\\r\n]|(\\\")|(\\\\))+</code> <var>(Quotes and<br>backslashes must be escaped with a backslash. A complex string must not<br>include newlines or carriage returns.)</var></a></pre>
```

File: files/en-us/web/api/xsltprocessor/index.html [hunk 16]
```diff
@@ -26,23 +26,23 @@ <h2 id="Methods">Methods</h2>
 
 <dl>
   <dt>
-    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> void </code>{{domxref("XSLTProcessor.importStylesheet")}}<code>(</code>{{domxref("Node")}}<code> styleSheet)</code>
+    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> void {{domxref("XSLTProcessor.importStylesheet")}}({{domxref("Node")}} styleSheet)</code>
```
Comment (Ryuno-Ki) [inline files/en-us/web/api/xsltprocessor/index.html:29]: What happened here?

File: files/en-us/web/api/xsltprocessor/index.html [hunk 17]
```diff
@@ -26,23 +26,23 @@ <h2 id="Methods">Methods</h2>
...
   </dt>
   <dd>Imports the XSLT stylesheet. If the given node is a document node, you can pass in a
     full XSL Transform or a <a class="external"
       href="https://www.w3.org/TR/xslt#result-element-stylesheet">literal result element
       transform</a>; otherwise, it must be an <code>&lt;xsl:stylesheet&gt;</code> or
     <code>&lt;xsl:transform&gt;</code> element.</dd>
   <dt>
-    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> </code>{{domxref("DocumentFragment")}}
-    {{domxref("XSLTProcessor.transformToFragment")}}<code>(</code>{{domxref("Node")}}<code> source, </code>{{domxref("Document")}}<code> owner)</code>
+    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> {{domxref("DocumentFragment")}}
```
Comment (Ryuno-Ki) [inline files/en-us/web/api/xsltprocessor/index.html:37]: This doesn't look right.
What make you change it like this?

File: files/en-us/web/api/xsltprocessor/index.html [hunk 18]
```diff
@@ -26,23 +26,23 @@ <h2 id="Methods">Methods</h2>
...
+    {{domxref("XSLTProcessor.transformToFragment")}}<code>(</code>{{domxref("Node")}} source, {{domxref("Document")}} owner)</code>
   </dt>
   <dd>Transforms the node source by applying the stylesheet imported using the
     {{domxref("XSLTProcessor.importStylesheet()")}} function. The owner document of the
     resulting document fragment is the owner node.</dd>
-  <dt><code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a></code>
+  <dt><code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a>
     {{domxref("Document")}}
-    {{domxref("XSLTProcessor.transformToDocument")}}<code>(</code>{{domxref("Node")}}<code> source)</code>
+    {{domxref("XSLTProcessor.transformToDocument")}}({{domxref("Node")}} source)</code>
```
Comment (Ryuno-Ki) [inline files/en-us/web/api/xsltprocessor/index.html:45]: See above.

File: files/en-us/web/api/xsltprocessor/index.html [hunk 19]
```diff
@@ -76,18 +76,18 @@ <h2 id="Methods">Methods</h2>
     </table>
   </dd>
   <dt>
-    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> void </code>{{domxref("XSLTProcessor.setParameter")}}<code>(</code>{{jsxref("String")}}<code> namespaceURI, </code>{{jsxref("String")}}<code> localName, any value)</code>
+    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> void {{domxref("XSLTProcessor.setParameter")}}({{jsxref("String")}} namespaceURI, {{jsxref("String")}} localName, any value)</code>
```
Comment (Ryuno-Ki) [inline files/en-us/web/api/xsltprocessor/index.html:79]: See above.

File: files/en-us/web/api/xsltprocessor/index.html [hunk 20]
```diff
@@ -76,18 +76,18 @@ <h2 id="Methods">Methods</h2>
...
   </dt>
   <dd>Sets a parameter in the XSLT stylesheet that was imported. (Sets the value of an
     <code>&lt;xsl:param&gt;</code>.) A null value for <code>namespaceURI</code> is treated
     the same as an empty string.</dd>
   <dt>
-    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> any </code>{{domxref("XSLTProcessor.getParameter")}}<code>(</code>{{jsxref("String")}}<code> namespaceURI, </code>{{jsxref("String")}}<code> localName)</code>
+    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> any {{domxref("XSLTProcessor.getParameter")}}({{jsxref("String")}} namespaceURI, {{jsxref("String")}} localName)</code>
```
Comment (Ryuno-Ki) [inline files/en-us/web/api/xsltprocessor/index.html:85]: See above.

File: files/en-us/web/api/xsltprocessor/index.html [hunk 21]
```diff
@@ -76,18 +76,18 @@ <h2 id="Methods">Methods</h2>
...
   </dt>
   <dd>Gets the value of a parameter from the XSLT stylesheet. A null value for
     <code>namespaceURI</code> is treated the same as an empty string.</dd>
   <dt>
-    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> void </code>{{domxref("XSLTProcessor.removeParameter")}}<code>(</code>{{jsxref("String")}}<code> namespaceURI, </code>{{jsxref("String")}}<code> localName)</code>
+    <code><a href="/en-US/docs/Mozilla/WebIDL_bindings#Throws">[Throws]</a> void {{domxref("XSLTProcessor.removeParameter")}}({{jsxref("String")}} namespaceURI, {{jsxref("String")}} localName)</code>
```
Comment (Ryuno-Ki) [inline files/en-us/web/api/xsltprocessor/index.html:90]: See above.

**Review by Ryuno-Ki — COMMENTED** [review]
This looks like you applied a regular expression on all files. It worked fairly well. I believe, NSS will go away somewhen soon, but the changes on xsltprocessor make me not approve it right away. Please explain in the comments.

Thank you for these changes.

**Comment by sideshowbarker** [comment]
@NilsIrl This PR seems to be waiting on responses from you to review comments/suggestions

**Comment by sideshowbarker** [comment]
Closing for lack of response

**Pull request closed without merging**